bundle-konflux.Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Workaround for https://issues.redhat.com/browse/STONEBLD-2496
+# This stages should be removed when the issue is resolved
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.22 as builder
```

Comment:
can we pull brew images in CI now?

Reply:
Not in CI, but I have given konflux a brew pull secret in order to workaround a lack of go 1.22 image in the "normal" build repositories.

Reply:
That was the recommended solution